### ISSUE TITLE
Request for dataset

### ISSUE THREAD

**Issue by OovessaliusoO** [WuJie1010/CitySCENE2020-Anomaly-Detection#1]
Dear WuJie,
    I'm interested in city anomaly detection recently,
    and I want to do some experiments on the dataset of CitySCENE Anomaly Detection Challenge,
    Would you like to share the dataset with me? 
    (I found the test set in the docker, but the training set is also necessary to me...)
